user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_organizing_landing_page.md

```diff
@@ -10,6 +10,9 @@ To do so:
 
 1. On the DMA, go to the app folder: `C:\Skyline DataMiner\applications\[APP_ID]`
 
+   > [!NOTE]
+   > To find our which app ID corresponds with which low-code app, check the URLs displayed in the browser when you open the apps via the DataMiner landing page.
```

Comment:
To find out*.
Would go with the singular approach when it comes to the URL (you can only see one URL/app at a time in one browser tab so it feels more natural).

Reply:
Adjusted!